Sources/Testing/Testing.docc/MigratingFromXCTest.md

```diff
@@ -355,6 +355,11 @@ their equivalents in the testing library:
 | `try XCTUnwrap(x)` | `try #require(x)` |
 | `XCTFail("…")` | `Issue.record("…")` |
 
+- Note: When comparing two numeric values within a specified accuracy, i.e. the
+analogue of [`XCTAssertEqual(_:_:accuracy:_:file:line:)`](https://developer.apple.com/documentation/xctest/3551607-xctassertequal)
+, please import [the swift-numerics package](https://github.com/apple/swift-numerics)
+and use the ``isApproximatelyEqual()`` function.
```

Comment:
```suggestion
to use the ``isApproximatelyEqual()`` function.
```

Reply:
Flip the whole ordering, which would make it more direct, something akin to:

"Import swift-numerics and use `isApproximateEqual` to compare two numeric values to within an accuracy bound you specify. Use this technique as a replacement for XCTAssertEqual(_:_:accuracy)."

or something akin

Reply:
That would be confusing since it would introduce instructions to import swift-numerics without a reader knowing _why_ that was important. State the problem before the solution.

Reply:
Yeah, you’re right. maybe lead with “swift-testing doesn’t include an exact duplicate of XCTAssertEqual(::accuracy)” and then jump into the direct advice.